tracee/tracee.bpf.c

```diff
@@ -26,8 +26,8 @@
 
 #undef container_of
 //#include "bpf_core_read.h"
-#include <bpf/bpf_helpers.h>
-#include <bpf/bpf_tracing.h>
+#include <bpf_helpers.h>
```

Comment:
Does compilation with the regular make works after this change?

Reply:
yes